src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -209,8 +222,14 @@ public static class SnackbarVisualElementExtension
 		this VisualElement? visualElement,
 		string message,
 		Action? action = null,
-		string actionButtonText = "OK",
+		string actionButtonText = Snackbar.defaultActionButtonText,
 		TimeSpan? duration = null,
 		SnackbarOptions? visualOptions = null,
-		CancellationToken token = default) => Snackbar.Make(message, action, actionButtonText, duration, visualOptions, visualElement).Show(token);
+		CancellationToken token = default)
+	{
+		ArgumentNullException.ThrowIfNull(message);
```

Comment:
Do we need it as we have a validation in property?

Reply:
> Do we need it as we have a validation in property

No, but I added it here for another layer of protection. You know how I like to bake in redundancy in our codebase 😀 

If anything changes in the future, to `DisplaySnackbar`, `Snackbar.Make`, `Snackbar.Text` or `Snackbar.ActionButtonText` in the future, I want to ensure that have multiple points of null validation. 

This is also why I explicitly added null validation for `DisplaySnackbar` in the Unit Tests.
```cs
	[Fact]
	public void SnackbarNullValuesThrowArgumentNullException()
	{
#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
		Assert.Throws<ArgumentNullException>(() => new Snackbar { Text = null });
		Assert.Throws<ArgumentNullException>(() => new Snackbar { ActionButtonText = null });
		Assert.Throws<ArgumentNullException>(() => Snackbar.Make(null));
		Assert.Throws<ArgumentNullException>(() => Snackbar.Make(string.Empty, actionButtonText: null));
		Assert.ThrowsAsync<ArgumentNullException>(() => new Button().DisplaySnackbar(null));
		Assert.ThrowsAsync<ArgumentNullException>(() => new Button().DisplaySnackbar(string.Empty, actionButtonText: null));
#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
	}
```

Reply:
Long story short, we may get a PR from a community member in the future who isn't familiar with the complete architecture of `Snackbar`, and this'll make sure that they don't accidentally break anything.

Reply:
As the property always used by the class (more often than the static method), I believe this is redundant. Also you will still get an exception if you call this method.

And something change in Text property (e.g. it became nullable) then we do not need this check in the Make method as well.

from my point of view identical logic on different layers leads to complexity and possible bugs

Reply:
Fair enough! I'll keep the null-check on the property and remove it from the method 👍 

The unit tests I added will protect us from any future PR breaking things